src/sidebar/util/dom.js

```diff
@@ -4,7 +4,8 @@
  *
  * @param {Element} element - DOM element to measure
  * @return {number|null} - The element's height in pixels or `null` if no
- *                         element with id `elementId` exists
+ *                         element with id `elementId` exists. Height is rounded
```

Comment:
This comment is out of date now because an `Element` is being passed in rather than an ID.